### ISSUE TITLE
Tune Virtual Memory Settings on Brokers

### ISSUE THREAD

**Issue by chuck-confluent** [confluentinc/cp-ansible#101]
Suggest tuning virtual memory settings in `/etc/sysctl.conf`:

```
vm.swappiness=1
vm.dirty_background_ratio=5
vm.dirty_ratio=60
```

These are recommended production settings for Brokers.

Thanks!

**Comment by JumaX** [comment]
@chuck-confluent we have just merged in the following PR which addresses this:

https://github.com/confluentinc/cp-ansible/pull/124

**Comment by chuck-confluent** [comment]
We might consider changing vm.dirty_ratio to 80, since that's what we apparently use internally for confluent cloud: https://confluentinc.atlassian.net/browse/CIRE-2480 . I tried to find other sources for optimal settings in the knowledge base and Scio but had little luck

**Comment by JumaX** [comment]
@domenicbove @chuck-confluent Can we open a new Issue/PR for this, as I don't think it's generally best practice to re-open old PRS to up make a new update.

**Comment by JumaX** [comment]
@chuck-confluent Just made the changed and merged it into 6.0.0-post.  Thanks again for the feedback.
